templates/Datashades-OpsWorks-CKAN-Instances.cfn.yml.j2

```diff
@@ -178,19 +262,21 @@ Resources:
           "{{ hour }}": "on"
 {% endfor %}
 {% endif %}
-      Hostname: !Sub "${ApplicationId}-web1"
+
+  #web1 OpswxEC2Web1PM or OpswxEC2Web1AO
+  OpswxEC2Web1{% if item.tags["PowerManaged"] == "Yes" %}PM{% else %}AO{% endif %}:
```

Comment:
This should be unchanged for non-power-managed instances so it doesn't shut down PRD.

Reply:
per line 307 comments, 24/7 -> 24/7 instance's does not remove old instances until all new instances are spun up.  (this is why unique hostnames is required) 

what if hostname is changed but not resource name:
well sadly there is a side effect of changing hostnames in that cloudformation 'won't' do it one after another, it will turn off all instances at once and then update hostname and boot up again (causing 5-20minutes of downtime)